.github/workflows/wee-slack.yml

```diff
@@ -0,0 +1,97 @@
+name: wee-slack
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: [2.7, 3.7]
```

Comment:
Does it make sense to lint with multiple Python versions? I agree on the test part, though :slightly_smiling_face:

Reply:
Yes. pyflakes will report different undefined names for different versions.

Reply:
At a fundamental level, it's useful to have this step use multiple versions (I'll even add 3.8 and 3.9 in a future CL), because different versions of the underlying tool may be installed and thus behave differently.

By running tools like flake8, black (TBD), and yamllint (TBD) in this step with this matrix, we ensure the code that exists in this repository works with the version of the tools which can be installed for any given `python_version`.

Reply:
At a fundamental level, it's useful to have this step use multiple versions (I'll even add 3.8 and 3.9 in a future CL), because different versions of the underlying tool may be installed and thus behave differently.

By running tools like flake8, black (TBD), and yamllint (TBD) in this step with this matrix, we ensure the code that exists in this repository works with the version of the tools which can be installed for any given `python_version`.